Python/quicc_solver/model/boussinesq/sphere/tc/explicit/physical_model.py

```diff
@@ -176,6 +180,8 @@ def convert_bc(self, eq_params, eigs, bcs, field_row, field_col):
                         bc = {0:-12, 'rt':1}
                     elif field_col == ("velocity","pol"):
                         bc = {0:-21, 'rt':2}
+                    elif field_col == ("temperature",""):
+                        bc = {0:-11, 'rt':1}
```

Comment:
is `0:-11` a range of modes? perhaps it would be better to have a named variable for the upper limit?

Reply:
No, it's not a range. 0 is a dictionary key (I don't remember why I chose that). and -11 is the ID. They should be replaced by proper IDs

Reply:
I see, ok